libs/deal-ticket/src/components/deal-ticket/deal-ticket-market-amount.tsx

```diff
@@ -46,7 +46,7 @@ export const DealTicketMarketAmount = ({
       <div className="flex items-end gap-4 mb-2">
         <div className="flex-1 text-sm">Size</div>
         <div />
-        <div className="flex-1 text-sm text-right">
+        <div className="flex-2 text-sm text-right">
```

Comment:
unwrap `Estimated uncrossing price`

Reply:
To me it looks nicer if the text is contained to an equal portion of the flex container, even if it wraps. If there is enough space it will unwrap.

Below the text flowing outside of the 'grid' and into the left sides space looks off to me as it disrupts the alignment the inputs below

![Screen Shot 2022-11-23 at 19 21 11](https://user-images.githubusercontent.com/6803987/203686718-7fc74bfd-521e-4ace-aaa4-321b46e8b9f9.jpg)